samples: sensors: nucleo_f756zg added to die_temp_polling

noahp opened the pull request:
Added board overlay file to permit internal temperature sensor testing:

- `nucleo_f756zg`

Signed-off-by: Noah Pendleton <noah.pendleton@gmail.com>

MaureenHelm commented on samples/sensor/die_temp_polling/boards/nucleo_f756zg.overlay:
```diff
@@ -0,0 +1,16 @@
+/ {
```
Please add a license/copyright header

noahp replied:
Done, thank you!

noahp committed:
samples: sensors: nucleo_f756zg added to die_temp_polling

Added board overlay file to permit internal temperature sensor testing:

- `nucleo_f756zg`

Signed-off-by: Noah Pendleton <noah.pendleton@gmail.com>
